gradle/license-report-config/allowed-licenses.json

```diff
@@ -63,6 +63,9 @@
     {
       "moduleLicense": "IAIK of Graz University of Technology License",
       "moduleName": "org.xipki.iaik:sunpkcs11-wrapper"
+    },
+    {
```

Comment:
Is this needed? It doesn't seem to define an allowed license

Reply:
yes, a peculiarity of the license plugin we are using. Since it can't determine the type of license, we add the license name in allowed list. Then we add it in override list to use our supplied license name.